instructions/1-your-first-component.md

```diff
@@ -236,7 +236,7 @@ export const Counter = React.createClass({
     return (
       <div>
         <h2>Count: {this.state.count}</h2>
-        <button type="button" onClick={this.incrementCounter}>increment</button>
+        <button type="button" onClick={() => this.incrementCounter()}>increment</button>
```

Comment:
Actually, you don't need a closure here as `React.createClass` will auto-bind all classes functions. By doing so you will avoid some potential performance issues.